### ISSUE TITLE
Suggestion for multiples of same string

### ISSUE THREAD

**Issue by JalonSolov** [vlang/v#4489]
Perl has a "nifty" feature of allowing you to type 'a' * 10 to get 10 'a' characters.

This would be immediately handy in V for things like tab indents, which are in 3 places right now, and all of them have a different, LIMITED number of tabs.

vlib/v/gen/cgen.c:

`        tabs = ['', '\t', '\t\t', '\t\t\t', '\t\t\t\t', '\t\t\t\t\t', '\t\t\t\t\t\t', '\t\t\t\t\t\t\t', '\t\t\t\t\t\t\t\t']`

vlib/v/gen/js/js.v:

`         tabs = ['', '\t', '\t\t', '\t\t\t', '\t\t\t\t', '\t\t\t\t\t', '\t\t\t\t\t\t', '\t\t\t\t\t\t\t', '\t\t\t\t\t\t\t\t']`

vlib/v/fmt/fmt.v:

`     tabs    = ['', '\t', '\t\t', '\t\t\t', '\t\t\t\t', '\t\t\t\t\t', '\t\t\t\t\t\t', '\t\t\t\t\t\t\t']`

Then each of these has code like the following (from fmt.v):

`    f.out.write(tabs[f.indent])`

However, if V had the same sort of character duplication (well, really string duplication) built-in, that line could be written as

`    f.out.write('\t' * f.indent)`

and not only would the const declaration of 'tabs' no longer be needed, it wouldn't by limited by the tedium of someone having to type in successively longer strings of '\t' characters.

The other advantage is that if it was ever decided that 4 spaces was better than a tab, the line above could just be changed to

`    f.out.write("    " * f.indent)`

Of course, the indent string would most likely be a const shared amongst the files, so it would only need to be changed in one place.  So somewhere has

```
const (
        common_indent = '\t'
)
```

And the code would be

`    f.out.write(common_indent * f.indent)`

This has a lot more uses than indenting code.  For example, I recently had to create a dummy SHA256 value, which is 64 characters long.  Rather than tediously counting to 64 while hitting a key, my Perl code looked like

`my $dummy_sha256 = 'a'*64;`

Basically, anywhere that you would need one or more characters repeated could be simplified with this feature.

**Comment by RNU4** [comment]
The syntax ```'a'*64``` seems unclear as your using a string with an int.
for example:
```
x:="a"

println(x*20) 
```
Maybe something like 'a'.repeat(20) like in golang
```
x:="a"

println(x.repeat(20)) //this is a lot clearer
```

**Comment by JalonSolov** [comment]
I am fine with .repeat() - I was just looking for the functionality, and using an example from another language.

**Comment by RNU4** [comment]
string.repeat is apparently already a thing
```
mut a := "a"
println(a.repeat(20))
```
output:
aaaaaaaaaaaaaaaaaaaa

**Comment by JalonSolov** [comment]
Then I withdraw my suggestion.
